src/main/java/app/studnicki/ox/Sign.java

```diff
@@ -0,0 +1,16 @@
+package app.studnicki.ox;
+
+public enum Sign {
```

Comment:
Hatred for Javadocs or Javadoc-phobia? Even with both, you still will need to write some. :P